cadc-storage-adapter-fs/src/main/java/org/opencadc/inventory/storage/fs/FileSystemStorageAdapter.java

```diff
@@ -317,40 +352,65 @@ public StorageMetadata put(NewArtifact newArtifact, InputStream source)
         InventoryUtil.assertNotNull(FileSystemStorageAdapter.class, "artifact", newArtifact);
         InventoryUtil.assertNotNull(FileSystemStorageAdapter.class, "source", source);
         
-        Path path = null;
-        Path existing = null;
+        Path txnPath = null;
+        Path contentPath = null;
```

Comment:
There should be usable contentPath and transactionPath variables setup in the constructor to use to use... don't recreate it here

Reply:
ok